dm/proto/dmworker.proto

```diff
@@ -166,6 +166,8 @@ message SyncStatus {
     int64 totalRows = 15;
     int64 totalRps = 16;
     int64 recentRps = 17;
+    uint64 ioTotalBytes = 18; // meter TCP io to downstream of the subtask 
+    uint64 dumpIOTotalBytes = 19; // meter TCP io from upstream of the subtask
```

Comment:
move comments above the line, else the generated `.pb.go` won't have them

Reply:
add test in https://github.com/pingcap/tiflow/pull/11742/commits/e9f348f07da921fdba8fef5dcc39ec3035de2dba